internal/backends/backend.go

```diff
@@ -88,7 +88,8 @@ type StatusParams struct{}
 
 // StatusResult represents the results of Backend.Status method.
 type StatusResult struct {
-	CountCollections int64
+	CountCollections       int64
+	CountCappedCollections int64
```

Comment:
This can be calculated by handler if needed. But since `CountCollections` is there too, I added it to StatusResult.